error:expected error

When I try to build flang, there is a error expected error which is in runtime/flang/CMakeFiles/flang_static.dir/build.make.
More details follow:

**/tmp/iso_c_bind-a0d584.ll:10:65: error: expected type
%structiso_c_binding___c_funptr__td_ = type < { [8 x i64], [6 x ptr], [28 x i8] } >
                                                                ^
1 error generated.
make[2]: *** [runtime/flang/CMakeFiles/flang_static.dir/build.make:3517: runtime/flang/CMakeFiles/flang_static.dir/iso_c_bind.F95.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/tmp/ftnhdr-87fa28.ll:4:25: error: expected type
define float @ftn_cabs_(ptr %c) !dbg !16 {
                        ^
1 error generated.
make[2]: *** [runtime/flang/CMakeFiles/flang_static.dir/build.make:766: runtime/flang/CMakeFiles/flang_static.dir/ftnhdr.F.o] Error 1
/tmp/ieee_features-562ea2.ll:10:75: error: expected type
%structieee_features__ieee_features_type___td_ = type < { [8 x i64], [6 x ptr], [38 x i8] } >
                                                                          ^
1 error generated.
make[2]: *** [runtime/flang/CMakeFiles/flang_static.dir/build.make:3476: runtime/flang/CMakeFiles/flang_static.dir/ieee_features.F95.o] Error 1
/tmp/omp_lib-58653b.ll:13:28: error: expected type
define void @omp_lib_oss1_(ptr %kind, ptr %modifier) !dbg !17 {
                           ^
1 error generated.
make[2]: *** [runtime/flang/CMakeFiles/flang_static.dir/build.make:1773: runtime/flang/CMakeFiles/flang_static.dir/omp_lib.F95.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:867: runtime/flang/CMakeFiles/flang_static.dir/all] Error 2
make: *** [Makefile:136: all] Error 2**


It would be great if someone can check it.

I try again，however， the scripts tries to copy flang2  to /usr/lib instead of install_prefix. It makes me confused！
More details fellow：
**Installing: /usr/lib/llvm-6.0/bin/flang2
CMake Error at tools/flang2/flang2exe/cmake_install.cmake:54 (file):
  file INSTALL cannot copy file
  "/home/lxj/flang9.0/flang-legacy/build/flang/bin/flang2" to
  "/usr/lib/llvm-6.0/bin/flang2": Success.
**

When I run build-flang.sh via source build-flang.sh，file INSTALL tries to copy flang2 to another while flang1 is installed correctly.
More details fellow：
**
-- Installing: /home/lxj/flang/install/lib/libflangmain.a
-- Installing: /home/lxj/flang/install/bin/flang1
-- Installing: /usr/lib/llvm-6.0/bin/flang2
CMake Error at tools/flang2/flang2exe/cmake_install.cmake:54 (file):
  file INSTALL cannot copy file
  "/home/lxj/flang/flang-legacy/build/flang/bin/flang2" to
  "/usr/lib/llvm-6.0/bin/flang2": Success.
Call Stack (most recent call first):
  tools/flang2/cmake_install.cmake:49 (include)
  tools/cmake_install.cmake:49 (include)
  cmake_install.cmake:58 (include)
**

--- a/tools/flang2/flang2exe/CMakeLists.txt
+++ b/tools/flang2/flang2exe/CMakeLists.txt
@@ -160,7 +160,7 @@ target_include_directories(flang2
 
 # Install flang2 executable
 install(TARGETS flang2
-        RUNTIME DESTINATION ${LLVM_TOOLS_BINARY_DIR})
+        RUNTIME DESTINATION bin)
 
 # Local Variables:
 # mode: cmake

